src/loaders/jobs/taskfarmer/taskfarmer_task_mgr.py

```diff
@@ -36,8 +38,9 @@ class JobStatus(Enum):
     PENDING = 'PENDING'
     RUNNING = 'RUNNING'
     COMPLETED = 'COMPLETED'
-    FAILED = 'FAILED'
+    FAILED = 'FAILED/CANCELLED'  # in many situations, we cannot identify whether a job has failed or cancelled
```

Comment:
Even with the slurm log it's impossible to tell if the job failed vs cancelled? There's no exit code? it seems surprising that slurm doesn't log that the job was cancelled somewhere

Reply:
yea. they both just showed 'KILLED' in the slurm log. Maybe as I do more testing, I will come up with some ways to identify those. But I think we could treat canceled as failed or vice visa at this time just fine.

Reply:
Weird that cancellation due to timeout is logged clearly but regular cancellation is indistinguishable from failure